Source/WebCore/dom/Document.cpp

```diff
@@ -4485,6 +4485,8 @@ void Document::setRequiresTrustedTypes(bool required)
         return;
 
     frame->checkedScript()->setRequiresTrustedTypes(required);
```

Comment:
Do we need to call this when the pref is disabled?

Reply:
I think yes, in case of weird situations where the JSC flag is enabled but this one isn't you could get into odd situations otherwise.

Reply:
I don't think we need to handle strange combinations like that. Maybe add an assertion to check they both use the same value.

Reply:
I don't think we need to handle strange combinations like that. Maybe add an assertion to check they both use the same value.